ivy_models_tests/inceptionnet/test_inceptionnet.py

```diff
@@ -0,0 +1,47 @@
+import os
+import ivy
+import pytest
+import numpy as np
+from ivy_models_tests import helpers
+from ivy_models.inceptionnet import inceptionNet_v3
+import jax
+
+jax.config.update("jax_enable_x64", False)
+
+
+@pytest.mark.parametrize("batch_shape", [[1]])
+@pytest.mark.parametrize("load_weights", [False, True])
```

Comment:
could you please try the new test format (your existing googlenet should be a good ref), thanks!